jena-core/src/main/java/org/apache/jena/util/JenaXMLInput.java

```diff
@@ -146,9 +147,9 @@ public static XMLInputFactory initXMLInputFactory(XMLInputFactory xmlInputFactor
         // disable external entities (silently ignore)
         setXMLInputFactoryProperty(xmlInputFactory, XMLInputFactory.IS_SUPPORTING_EXTERNAL_ENTITIES, Boolean.FALSE);
 
-        // Not supported by Woodstox. IS_SUPPORTING_EXTERNAL_ENTITIES = false is enough.
+        // Not supported by Woodstox an Aalto. IS_SUPPORTING_EXTERNAL_ENTITIES = false is enough.
```

Comment:
```suggestion
        // Not supported by Woodstox or Aalto. IS_SUPPORTING_EXTERNAL_ENTITIES = false is enough.
```

Reply:
Thanks. I made a new commit and squashed it.